README.md

```diff
@@ -15,6 +15,8 @@ By invoking sheriff with specific options,
 those tags determine whether a field will be added to the output map or not. It
 can then be marshalled using "encoding/json".
 
+**NOTE**: Since this package uses the *context* package, it will only work on Go 1.7+
```

Comment:
hm, where do we use context?

Reply:
true, we don't directly, it's the build pipeline that uses goimport, which uses context. Probably we can change the notice so that is clear that sheriff is tested on go 1.7+, but should work even on older versions

Reply:
```
 **NOTE**: This package is tested only on Go 1.7+, it *might* work on Go 1.6 too, but no support is given.
```

something like that?

Reply:
Yes, thanks :)